src/backtrace/mod.rs

```diff
@@ -129,15 +129,15 @@ with_backtrace! {
     impl Debug for Backtrace {
         fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
             if let Some(bt) = self.internal.as_backtrace() {
-                bt.fmt(f)
+                Debug::fmt(bt, f)
             } else { Ok(()) }
         }
     }
 
     impl Display for Backtrace {
         fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
             if let Some(bt) = self.internal.as_backtrace() {
-                bt.fmt(f)
+                Debug::fmt(bt, f)
```

Comment:
@RalfJung I think this should be `Display`.

Reply:
That would require bumping the minimal version of backtrace, though.

And anyway I closed this PR because https://github.com/rust-lang-nursery/failure/pull/279 does the same thing.

Reply:
Right, yeah, I was trying to answer: https://github.com/rust-lang-nursery/failure/pull/279#issuecomment-446373879

But I didn't know about the minimal version requirement.